Gemfile

```diff
@@ -15,6 +15,6 @@ elsif ENV['RAILS'] == "4.1"
   gem 'activemodel', '~> 4.1.0'
   gem 'globalize', '~> 4.0'
 else # Rails 4.2
-  gem 'globalize', '~> 5.0'
-  gem 'paper_trail', '4.0.0.beta2'
+  gem 'globalize', '5.1.0.beta2'
```

Comment:
Do we really need to stick with particular (beta) version? This would lead to version incompatibility in many projects. Can we keep '~> 5.0', so that all 5.X versions would work?

Reply:
You're right, I just wanted to test if it works with the latest Globalize version. Unfortunately Globalize v5.0.1 is more than 1.5 years old...